simulator/src/api/marketapi.ts

```diff
@@ -13,7 +13,7 @@ var router = express.Router();
 /**
  * Get current market price.
  */
-router.post('/update/price', ensureAuthenticated('manager'), (req: express.Request, res: express.Response) => {
+router.put('/price', ensureAuthenticated('manager'), (req: express.Request, res: express.Response) => {
```

Comment:
You do not need express.Request/ express.Response type due to type inference.

Reply:
It still works, it is only forms in html that can't do put calls